src/appleseed/renderer/modeling/bsdf/plasticbrdf.cpp

```diff
@@ -398,7 +406,8 @@ namespace
             const Vector3f&         wo,
             const Vector3f&         m,
             const float             F,
-            Spectrum&               value)
+            Spectrum&               value,
+            const float             gamma)
```

Comment:
Makes complete sense, this was a leftover after removing the default values which required the variable to come at the end. Fixed!